assets/styl/_modules/_buttons.styl

```diff
@@ -106,37 +132,37 @@
         margin .33em
         display block
 
-//disabled & hover effect
+//disabled & hover default effect (for buttons without a key color)
 .btn
 .btn-icon
     &:disabled
-        background-color alpha($color-secondary, .5)
-        color alpha($color-base, .5)
+        opacity .5
 
     &:enabled:hover
         background-color darken($color-secondary, 20%)
 
 // btn theme color
 .btn
 .btn-icon
-    for key, value in $colors
+    for key, value in merge($colors, $colors-socials)
         &.btn-{key}
         &.btn-icon-{key}
             background-color value
             color $color-white
 
-            &:disabled
-                background-color alpha(value, .5)
-
             &:enabled:hover
                 background-color darken(value, 20%)
 
     &.btn-native
+    &.btn-snapchat
         color $color-base
 
-        &:disabled
-            color alpha($color-base, .5)
+    &.btn.btn-pocket
+    &.btn-icon.btn-pocket
```

Comment:
These selectors are weird: [see here how they output](https://github.com/20minutes/colette/pull/145/files#diff-14a4c489c8facdc2c10e78c1487881c4R1656).
Can't `.btn-pocket` do the job?